src/Data/Fin/Permutation/Components.agda

```diff
@@ -42,17 +34,31 @@ transpose i j k with does (k ≟ i)
 --  Properties
 ------------------------------------------------------------------------
 
+transpose-iij : ∀ {n} (i j : Fin n) → transpose i i j ≡ j
```

Comment:
Normal naming conventions would probably be `transpose[i,i,j]≡j`? Same for the three below?